whirlpool/eventsocket.py

```diff
@@ -125,7 +125,6 @@ async def _run(self):
                             continue
                         self._msg_listener("{" + match[0] + "}")
         except (
-            aiohttp.ClientConnectionError,
```

Comment:
Is this related to the unregister callback?

Reply:
No - but it was pointed out that the original ClientError is all that is needed as ClientConnectionError is under ClientError.  Thought I might as well clean it up.